apps/meteor/ee/server/api/sessions.ts

```diff
@@ -0,0 +1,169 @@
+import { Users, Sessions } from '@rocket.chat/models';
+import { IUser } from '@rocket.chat/core-typings';
+import { escapeRegExp } from '@rocket.chat/string-helpers';
+
+import { isSessionsPaginateProps, isSessionsProps } from '../../definition/rest/v1/sessions';
+import { API } from '../../../app/api/server/api';
+import { hasLicense } from '../../app/license/server/license';
+import { Notifications } from '../../../app/notifications/server';
+
+const validateSortKeys = (sortKeys: string[]): boolean => {
+	const validSortKeys = ['loginAt', 'device.name', 'device.os.name', 'device.os.version', '_user.name', '_user.username'];
+
+	return sortKeys.some((s) => validSortKeys.some((v) => s === v));
```

Comment:
in this case it should be a `every`, right? since `sortKeys` should have **only valid** keys..

I think we can avoid the second "loop" as well:
```suggestion
	return sortKeys.every((s) => validSortKeys.includes(s));
```